exercise_150514_Hangman.rb

```diff
@@ -0,0 +1,124 @@
+require "colorize"
+
+class Hangman
+
+  def initialize
+    @wrong_count = 0
+    @ascii = [
+      "|     _________".light_black,
+      "|     |/      |".light_black,
+      "|     |".light_black,
+      "|     |".light_black,
+      "|     |".light_black,
+      "|     |".light_black,
+      "|     |".light_black
+      ]
+     @man_array = [
+      "|     |      / \\".colorize(:red),
+      "|     |       |".colorize(:cyan),
+      "|     |      \\|/".colorize(:yellow),
+      "|     |      (_)".colorize(:magenta)
+    ]
+    @user_guesses = []
+
+    @word = pick_word
+    @underlines = create_underlines
+    puts "Answer: #{@word.join.upcase}" ##TODO: Remove this before it goes to the user.
+
+    @play_status = true
+    play
+  end
+
+  def pick_word
+    word_options = %w(scared selection shake shaking shallow shout silly simplest slight slip slope soap solar species spin stiff swung tales thumb tobacco toy trap treated tune university vapor vessels wealth wolf zoo)
+    word = word_options.sample.split(//)
+    return word
+  end
+
+  def create_underlines
+    underlines = ["| ____|___ word: ".light_black]
+    @word.length.times do
+      underlines.push("_ ")
+    end
+
+    return underlines
+  end
+
+  def ascii_art
+    @ascii[7] = @underlines.join
+    puts @ascii
+  end
+
+  def play
+    puts "Welcome! Let's play Hangman!".cyan
+    while true
```

Comment:
We originally had something... but in any sort of win/lose scenario, it never made it back to the start of the loop because we aborted the program when they won. Even if we set "play_status = false" before we ended the game, it wouldn't ever check it anyway.

I'm guessing, though, that it maybe would be a good idea to have that anyway in case something changed later?